packages/web3-core/src/web3_request_manager.ts

```diff
@@ -74,15 +75,19 @@ export class Web3RequestManager<
 }> {
 	private _provider?: SupportedProviders<API>;
 	private readonly useRpcCallSpecification?: boolean;
+	public config: Partial<Web3ConfigOptions>;
```

Comment:
web3 config is already always in web3-context so it should be used from there in other parts of lib, so considering that it should not be added on another level ( request manager ) https://github.com/web3/web3.js/blob/b3cb1b782cc2c62bd87909c7cdc426ed43a49c1c/packages/web3-core/src/web3_context.ts#L51